pkg/client/tests/remotecommand_test.go

```diff
@@ -188,8 +188,8 @@ func TestStream(t *testing.T) {
 	}
 
 	for _, testCase := range testCases {
-		t.Run(testCase.TestName, func(t *testing.T) {
-			for _, exec := range []bool{true, false} {
+		for _, exec := range []bool{true, false} {
+			t.Run(testCase.TestName, func(t *testing.T) {
```

Comment:
you have to move 
```

				var name string
				if exec {
					name = testCase.TestName + " (exec)"
				} else {
					name = testCase.TestName + " (attach)"
				}
```
out of the loop and use `name` instead of testCase.TestName in `t.Run`

Reply:
Done, thanks for pointing out. @aojea